test/suite/test_compat01.py

```diff
@@ -62,7 +64,8 @@ class test_compat01(wttest.WiredTigerTestCase, suite_subprocess):
         ('old_patch', dict(compat1="1.8.1", logv1=1)),
     ]
     restart_compat = [
-        ('def2', dict(compat2='none', logv2=3)),
+        ('def2', dict(compat2='none', logv2=4)),
+        ('100_0', dict(compat2='10.0', logv2=4)),
```

Comment:
I suggest naming it `100_2` as the list of names is references that it is "the second list" for test purposes.

Reply:
Done.